docs/terminal/basics.md

```diff
@@ -22,6 +22,23 @@ To open the terminal:
 
 > **Note:** Open an external terminal with the `kb(workbench.action.terminal.openNativeConsole)` keyboard shortcut if you prefer to work outside VS Code.
 
+## Opening terminals from the Explorer
```

Comment:
This doesn't seem like the most important thing in the document so it shouldn't be the first heading, even before explaining what a terminal shell is?

Can it move to a bullet point in the "To open the terminal" part instead? We could do a one liner there and move the keybinding part to the FAQ?

Reply:
Yep, it's a nice flow but I was struggling how to present without it's own section.
I'll switch to a bullet and add a FAQ entry